### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by callumacrae** [euvl/vue-js-toggle-button#9]
Hey!

This component currently doesn't work with screen readers. It seems a pretty easy fix - just make the checkbox element visible to screen readers.

Also, are you fine if I use this library as an example in a talk of an input library which doesn't work with screen readers? It 100% isn't intended as an attack and it won't be the only library I talk about - most libraries in the Vue ecosystem have the same problem!

**Comment by euvl** [comment]
Hey,

Sure feel free, just put a link to github repo somewhere as reference 😄 
I am not very familiar with `aria`,  it should be ` aria-checked="STATE" ` and thats it?

**Comment by callumacrae** [comment]
That should do the trick, I think. Happy to test any changes.

**Comment by euvl** [comment]
Feel, free "accessibility" branch
